src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1291,6 +1305,38 @@ local move_queue, lines = {}, {}
 local find_portal, vision_fail, room_detected, random_move, force_portal, find_prompt, downloading, walking, help_shown
 local mt = getmetatable(map) or {}
 
+local exitmap = {
```

Comment:
This is an (incomplete) copy of the local variable `exitmap` below. The other ones have been removed below, why not this one?